include/index_helper.cuh

```diff
@@ -1106,4 +1106,302 @@ namespace quda {
     return (((x[3]*X[2] + x[2])*X[1] + x[1])*X[0] + x[0]) >> 1;
   }
 
+
```

Comment:
Here in a namespace `openqcd`, I added all indexing helper function that are needed by the order classes to reduce code duplication.